cmd/flags_misc.go

```diff
@@ -14,8 +15,9 @@ import (
 )
 
 var (
-	logLevelFlag = "log.level"
-	pprofFlag    = "pprof"
+	logLevelFlag       = "log.level"
+	logLevelModuleFlag = "log.level.module"
```

Comment:
can't this just be `log.module`

Reply:
lets just leave as is. I never seen a flag with two `.` in it 😂  

If we have complaints, we can simplify.